### PR TITLE
Compatible to ROS Kinetic Kame

### PR DESCRIPTION
**Pull request by Voidminded**
Can be compiled with OpenCV 3.1.0 which is used by ROS Kinetic on Ubuntu 16.04
Used Eigen3 and Cpp-11 to suppress some warnings
Set CMake policy to remove dependencies warnings for CMake 3+

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,13 +9,11 @@ notifications:
   email:
     on_failure: always
     recipients:
-      - 130s@2000.jukuin.keio.ac.jp
+      - smohaime@sfu.ca
```
Comment (130s) [inline .travis.yml:12]: -1. You don't need to remove recipient addresses of someone else unless you're 100% sure.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: trusty
+dist: xenial
```
Comment (130s) [inline .travis.yml:2]: AFAIK, `Travis CI` doesn't yet handle Ubuntu Xenial. Instead, `industrial_ci` uses docker for testing ROS Kinetic on Xenial.

**Comment by 130s** [comment]
Thank you for the update. As I've updated already the `.travis.yml` in #79, you don't need the change you made in this PR.
But your OpenCV change is probably needed for ROS Kinetic as I also mentioned in #79.

@Voidminded My question is, does this build both for ROS Indigo-Jade on Truesty **AND** ROS Kinetic on Xenial?
- If so we can stick to a single branch (`indigo-devel`). 
- If not we might want to create a separate branch for kinetic.

**Comment by 130s** [comment]
> - If so we can stick to a single branch (indigo-devel). 

If this is the case, could you:
- remove the change you made in `.travis.yml`?
- rebase your branch?

> - If not we might want to create a separate branch for kinetic.

If this is the case, I'll create `kinetic-devel`.

**Comment by Voidminded** [comment]
ROS Kinetic is using OpenCV 3.x, the code is not compatible anymore with indigo-devel. So please create a new branch for Kinetic.

Also Travis is not supporting xenial yet, I think we should remove it or use other CI

**Comment by 130s** [comment]
Closing in favor of #80.

@Voidminded 

> Also Travis is not supporting xenial yet, I think we should remove it or use other CI

Please take a look at https://github.com/sniekum/ar_track_alvar/pull/79.

**Pull request closed without merging**